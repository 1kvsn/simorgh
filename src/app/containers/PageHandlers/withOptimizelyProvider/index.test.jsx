// import React from 'react';
// import {
//   shouldMatchSnapshot,
//   suppressPropWarnings,
// } from '@bbc/psammead-test-helpers';

// import WithOptimizelyProvider from '.';

// describe('withOptimizelyProvider HOC', () => {
//   beforeEach(() => {
//     jest.clearAllMocks();
//   });

//   const Component = () => <h1>Hola Optimizely</h1>;
//   const WithOptimizelyProviderHOC = WithOptimizelyProvider(Component);

//   it('should only render firstPublished timestamp for Igbo when lastPublished is less than 1 min later', async () => {});
// });
