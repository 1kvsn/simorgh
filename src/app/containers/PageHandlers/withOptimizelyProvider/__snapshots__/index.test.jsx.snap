// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withOptimizelyProvider HOC should load Optimizely with correct params 1`] = `
<div>
  <h1>
    Hola Optimizely
  </h1>
</div>
`;
